pyrsa/data/dataset.py

```diff
@@ -32,8 +33,8 @@ class DatasetBase:
         Returns:
             dataset object
     """
-    def __init__(self, measurements=None, descriptors=None,
-                 obs_descriptors=None, channel_descriptors=None):
+    def __init__(self, measurements, descriptors={},
+                 obs_descriptors={}, channel_descriptors={}):
```

Comment:
Using a mutable object like a `dict` or `list` for an argument default value is generally a bad idea, as the changed object will later server as the default again..

Reply:
That is definitely good to know. I did not expect that behavior here. Fixed in a new commit.